qemux86-64.robot

```diff
@@ -11,7 +11,7 @@ Preparing test environment
   Set Suite Variable    ${RESINRC_RESIN_URL}    %{RESINRC_RESIN_URL}
   Set Suite Variable    ${image}    %{image}
   File Should Exist     ${image}  msg="Provided images file does not exist"
-  Set Suite Variable    ${application_repo}    https://github.com/resin-io-projects/alpine-barebone.git
+  Set Suite Variable    ${application_repo}    https://github.com/villaseca/kernel-server.git
```

Comment:
I would prefer not to change this URL end point for each test - Maybe this warrants a new repository just for the on device test helper codebase?